src/components/SendFundsForm.vue

```diff
@@ -495,6 +502,27 @@ export default {
     },
     allowIncreaseFee () {
       return (this.currency === Cryptos.BTC) || isEthBased(this.currency)
+    },
+    currentCurrency: {
+      get () {
+        return this.$store.state.options.currentRate
+      },
+      set (value) {
+        this.$store.commit('options/updateOption', {
+          key: 'currentRate',
+          value
+        })
+      }
+    },
+    transferFeeRate () {
+      const currentRate = this.$store.state.rate.rates[`${this.transferFeeCurrency}/${this.currentCurrency}`]
+      const feeRate = (this.transferFeeFixed * currentRate).toFixed(2)
+      return currentRate !== undefined ? `${feeRate} ${this.currentCurrency}` : ''
```

Comment:
You can check if `currentRate` is undefined right after getting it from the store and immediately return the empty string. E.g.:

```js
const currentRate = this.$store.state.rate.rates[`${this.transferFeeCurrency}/${this.currentCurrency}`]

if (currentRate === undefined) {
  return ''
}

const feeRate = (this.transferFeeFixed * currentRate).toFixed(2)

return feeRate
```